invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -571,3 +571,10 @@ dl.details-list {
 .left-floated {
   float: left;
 }
+
+.separated:not(:last-child)::after {
+  content: ";";
```

Comment:
could be as a variable in site.variables
`@listSeparator: ";"`
for easy overriding